package/environments/development.js

```diff
@@ -14,10 +14,15 @@ let devConfig = {
 if (runningWebpackDevServer) {
   if (devServer.hmr) {
     devConfig = merge(devConfig, {
-      output: { filename: '[name]-[hash].js' }
+      output: { filename: '[name]-[contentHash].js' }
```

Comment:
I am not sure if contentHash should be used.
As #3060 skips `contentHash` for non prod environments.

Reply:
Right now it's bringing up a deprecation warning. This at least eliminates that while maintaining the existing functionality.

It's also possible that something down the line is overriding this anyway...